routes/api.php

```diff
@@ -31,6 +31,10 @@
             Router::get('users', 'Administration\UsersController@index');
             Router::get('users/{user}', 'Administration\UsersController@get');
 
+            //User profile
+            Router::get('admin/{user}/profile', 'Administration\UsersController@profile');
+            Router::put('admin/{user}/profile', 'Administration\UsersController@update');
+
```

Comment:
The logged in use should be able to directly poll the api end point without an id. 

ie: api/1.0/admin/profile

These other routes should be the same. This is different from the admin/user end points.

Reply:
Changed